web/04-runpecan.php

```diff
@@ -68,7 +68,14 @@
 if (!isset($_REQUEST['ensemble'])) {
   die("Need an ensemble value.");
 }
-$ensemble=$_REQUEST['ensemble'];
+$ensemble = $_REQUEST['ensemble'];
+
+$sensitivity = array();
+if (!isset($_REQUEST['sensitivity'])) {
+  die("Need a sensitivity value-comma seperated.");
```

Comment:
I agree that this is unclear but @dlebauer I don't think your example actually makes this more clear. Also, I don't think these are good defaults (default should be that the SA is turned off). Looking at the code below it seems that what's expected is actually sigma equivalents, which is OK but we do need the ability to enter quantiles as well.  This design is an OK first pass (and I'm happy to pull it once there's more explanation about what's expected), but ultimately entering a comma-separated list doesn't seem like the best design. On one hand, it's going to get really annoying to have to type in if you're doing a lot of this. On the other hand, something more graphical might provide a more intuitive method for specifying quantile/sigma points.

More generally, I didn't realize until I pulled the latest code that the Ensembles and SA are being entered on a completely new page that was a default behavior. I think the default behavior should be to skip this menu entirely, and only go here if you select some sort of "Additional Settings" box or something like that.  Right now we're getting a lot of stuff creeping into the UI that really doesn't have to be there for most runs and that's making the overall design less clean. We're really overdue to sit down and discuss what we want the UI to look like, how we want it to function, and how we want to accommodate features we know are coming down the pipeline.